rllib/agents/dqn/simple_q.py

```diff
@@ -0,0 +1,87 @@
+import logging
+
+from ray.rllib.agents.trainer import with_common_config
+from ray.rllib.agents.dqn.simple_q_tf_policy import SimpleQTFPolicy
+from ray.rllib.agents.dqn.dqn import DQNTrainer
+
+logger = logging.getLogger(__name__)
+
+# yapf: disable
+# __sphinx_doc_begin__
```

Comment:
Gave SimpleQ its own config. Seemed confusing to have it inherit from DQN first, then "downgrade" DQN's config to be "simple".